File: src/Clients/RavenSMS.Client.Ionic/src/app/core/constants/enums/DisconnectionReason.ts
/**
 * an enum that defines the disconnection reasons
 */
export enum DisconnectionReason {
    Disconnect = 'disconnect',
    ClientDeleted = 'client_deleted',
    ClientNotFound = 'client_not_found',
    ClientAlreadyConnected = 'client_already_connected'
}

File: src/RavenSMS/Core/Constants/DisconnectionReason.cs
namespace RavenSMS.Internal.Constants;

/// <summary>
///  class that defines the disconnection reasons
/// </summary>
internal static class DisconnectionReason
{
    public const string Disconnect = "disconnect";
    public const string ClientDeleted = "client_deleted";
    public const string ClientNotFound = "client_not_found";
    public const string ClientAlreadyConnected = "client_already_connected";
}
